volatility3/framework/plugins/windows/pslist.py

```diff
@@ -206,35 +208,37 @@ def filter_func(x):
     def list_processes(
         cls,
         context: interfaces.context.ContextInterface,
-        layer_name: str,
-        symbol_table: str,
+        kernel_module_name: str,
         filter_func: Callable[
             [interfaces.objects.ObjectInterface], bool
         ] = lambda _: False,
     ) -> Iterator["extensions.EPROCESS"]:
-        """Lists all the processes in the primary layer that are in the pid
+        """Lists all the processes in the given layer that are in the pid
         config option.
 
         Args:
             context: The context to retrieve required elements (layers, symbol tables) from
-            layer_name: The name of the layer on which to operate
-            symbol_table: The name of the table containing the kernel symbols
+            layer_iname: The name of the layer on which to operate
+            symbol_table_name: The name of the table containing the kernel symbols
             filter_func: A function which takes an EPROCESS object and returns True if the process should be ignored/filtered
 
         Returns:
             The list of EPROCESS objects from the `layer_name` layer's PsActiveProcessHead list after filtering
         """
 
+        kernel = context.modules[kernel_module_name]
+
         # We only use the object factory to demonstrate how to use one
-        kvo = context.layers[layer_name].config.get("kernel_virtual_offset", None)
+        kvo = context.layers[kernel.layer_name].config.get(
```

Comment:
I think we can just check `kernel.offset` rather than looking up the `kvo`?  It's fine to leave it this way, but we'll probably rip it out as a mechanism down the road.